source/util/KeyDecoder.cpp

```diff
@@ -0,0 +1,21 @@
+#include <radix/util/KeyDecoder.hpp>
+
+#include <SDL_scancode.h>
+
+namespace radix {
+  
+const std::map<std::string, int> KeyDecoder::lookupTable = { 
+  {"escape", SDL_SCANCODE_ESCAPE},
+  {"return", SDL_SCANCODE_RETURN},
+  {"space", SDL_SCANCODE_SPACE},
+  {"up", SDL_SCANCODE_UP},
+  {"down", SDL_SCANCODE_DOWN},
+  {"left", SDL_SCANCODE_LEFT},
+  {"right", SDL_SCANCODE_RIGHT},
+};
+
+int KeyDecoder::decodeString(const std::string &keyString) {
+  return lookupTable.find(keyString)->second;
```

Comment:
@Geertiebear What if the string to decode is not in the table? Can we throw an exception in that case?

https://stackoverflow.com/questions/1939953/how-to-find-if-a-given-key-exists-in-a-c-stdmap#1939971